docs/source/model_doc/tapas.rst

```diff
@@ -142,10 +161,21 @@ way. Here's an example:
         >>> from transformers import TapasConfig, TapasForQuestionAnswering
 
         >>> # you can initialize the classification heads any way you want (see docs of TapasConfig)
-        >>> config = TapasConfig(num_aggregation_labels=3, average_logits_per_cell=True, select_one_column=False)
+        >>> config = TapasConfig(num_aggregation_labels=3, average_logits_per_cell=True)
```

Comment:
Is there a reason you omitted this from the code example?

Reply:
Yes. Invalid combination of config parameters. with `select_one_column=False` model forward throw error. 
![Screenshot 2021-09-16 at 6 51 27 PM](https://user-images.githubusercontent.com/17096858/133619987-b24d2a3f-b971-4d8a-a6c2-da09f2a164ea.png)